theme/recipes/button-recipe.ts

```diff
@@ -74,6 +73,12 @@ export const buttonRecipe = defineRecipe({
         _active: {
           bg: 'accent.component-background-pressed',
         },
+        _disabled: {
+          _hover: { bg: 'unset' },
+          border: '1px solid {colors.accent.non-interactive}',
```

Comment:
Maybe we can add a border token for this?

Reply:
https://github.com/leather-wallet/mono/issues/48